### DISCUSSION TITLE
patch patch-commit

### DISCUSSION THREAD

**Discussion by xinbingyao** [pnpm/pnpm#5098] [Q&A] [answered]
```javasript
ERR_PNPM_PATCH_NOT_APPLIED  The following patches were not applied: tinymce@6.1.0

Either remove them from "patchedDependencies" or update them to much packages in your dependencies.

```
When I execute `pnpm install` again after `patch-commit`, this error will appear.
What should I do to add the modified file to node_ modules.
Thank you!

**Comment by zkochan** [comment, marked as answer]
Update the version of `tinymce@6.1.0` to the one that is installed
